tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -10658,6 +10658,10 @@
      "192c7235d061b439ef2b57d4b01b170b7412dcdc",
      []
     ],
+    "navigate_loop_crash.html": [
```

Comment:
Name this `navigate_loop-crash.html`: https://web-platform-tests.org/writing-tests/crashtest.html#crashtest-tests